libs/domain/cart/quantity-input/index.ts

```diff
@@ -1 +1,2 @@
+/** @deprecated since 1.2. Use quantity input from @spryker-oryx/ui/quantity-input*/
```

Comment:
Please check everywhere.
```suggestion
/** @deprecated since 1.2. Use oryx-quantity-input from @spryker-oryx/ui/quantity-input */
```